.github/workflows/tests.yml

```diff
@@ -82,6 +82,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install tox
+          echo "TOX_MAJOR=$(tox --version | awk '{split($0, s,"."); print s[1]}')" >> $GITHUB_ENV
```

Comment:
I would do the full version instead of just the major part. having to recreate the cache a bit more often is fine for me, and should help preventing this kind of issues later.

Reply:
fine for me.